src/lib/x509/certstor_system_windows/info.txt

```diff
@@ -0,0 +1,15 @@
+<defines>
+CERTSTOR_WINDOWS -> 20190430
+</defines>
+
+<os_features>
+win32
+</os_features>
+
+<header:public>
+certstor_windows.h
+</header:public>
+
+<libs>
+all!windows -> crypto
```

Comment:
Is this for Wine?

Reply:
Uhm ... I stole this from `src\lib\prov\openssl\info.txt` 
I think it didn't work before, but worked after ... it's entirely possible that another change fixed it though. I can check if the `all!` is still necessary.

Reply:
On Linux/etc -lcrypto == openssl
It looks like Crypt32.lib is needed for the Cert APIs so I think this should just be
windows -> crypt32